### PR TITLE
upgrade Guava

### PR DESCRIPTION
**Pull request by lesv**
Fix dip testing

### CODE REVIEW

**Comment by lesv** [comment]
@jabubake FYI

File: dlp/src/test/java/com/example/dlp/InspectIT.java [hunk 1]
```diff
@@ -44,7 +44,7 @@
   public void setUp() {
     bout = new ByteArrayOutputStream();
     out = new PrintStream(bout);
-    System.setOut(out);
+    System.setOut(out); // TODO(b/64541432) DLP currently doesn't support GOOGLE DEFAULT AUTH
```
Comment (jabubake) [inline dlp/src/test/java/com/example/dlp/InspectIT.java]: Les, can we refer to the [Github issue](https://github.com/GoogleCloudPlatform/google-cloud-java/issues/2231) instead : so users can view details ?

File: vision/cloud-client/pom.xml [hunk 2]
```diff
@@ -50,7 +50,7 @@
     <dependency>
       <groupId>com.google.guava</groupId>
       <artifactId>guava</artifactId>
-      <version>20.0</version>
+      <version>23.0</version>
```
Comment (jabubake) [inline vision/cloud-client/pom.xml]: Is there a reason we went for 23.0 here and 22.0 in guestbook on J8 ?

**Comment by lesv** [comment]
Guava needs to be set manually.  I think I used 23 where it worked.  In the case of DLP, it seemed like where the issue needed to go wasn't on Github.

**Pull request merged**